packages/manager-components/package.json

```diff
@@ -83,11 +82,11 @@
     "storybook": "7.5.3",
     "storybook-addon-react-router-v6": "^2.0.10",
     "ts-jest": "^29.1.1",
-    "typescript": "^4.3.2",
-    "vite": "4.4.9",
+    "vite": "^5.2.13",
     "vite-plugin-dts": "3.5.1",
     "vitest": "0.34.1",
-    "zustand": "^4.5.0"
+    "zustand": "^4.5.0",
+    "@ovh-ux/manager-vite-config": "^0.6.2"
```

Comment:
**nitpick:** This should be in the correct sorting order. Strange that it was not auto-corrected by `sort-package-json`.

Reply:
👍